src/main/java/com/khutircraftubackend/services/registration/RegistrationSellerController.java

```diff
@@ -0,0 +1,34 @@
+package com.khutircraftubackend.services.registration;
+
+import com.khutircraftubackend.dto.SellerDTO;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/v1/api")
+@RequiredArgsConstructor
+public class RegistrationSellerController {
+
+    private final RegistrationSellerService registrationSellerService;
+
+    @PostMapping("/registerSeller")
```

Comment:
taking in account comment above about the request mapping ,
this should have been `/register` which is under `/v1/seller`

Reply:
I wanted to separate ways with this name. It's okay.